core/templates/dev/head/pages/topic_editor/questions/QuestionsTabDirective.js

```diff
@@ -62,8 +84,9 @@ oppia.directive('questionsTab', [
               $scope.skillId, $scope.question.toBackendDict(true)
             ).then(function() {
               TopicEditorStateService.fetchQuestionSummaries(
-                $scope.topic.getId()
+                $scope.topic.getId(), true
```

Comment:
So after testing some more, it seems like the questions are not being surfaced in any particular order. I think that ideally the creator can see the question in the list right after creating it so that they have some confirmation. Also more generally, there should probably be some sort of order for viewing the questions, or else it may be hard to navigate.

Do you think you could sort by the "question_model_last_updated" field of QuestionSummaryModel in reverse chronological order? This way, it makes sense to reset the cursor to the first page, because the newly created question should be the very first entry.

Reply:
Done, sorting by last updated now, such that the latest updated is at the top.